TF_Service_dotNet/TouchFree/Interactions/InteractionModules/InteractionModule.cs

```diff
@@ -73,36 +95,19 @@ public void Update()
             {
                 handChirality = hand.IsLeft ? HandChirality.LEFT : HandChirality.RIGHT;
 
-                positions = positioningModule.CalculatePositions(hand);
+                positions = positioningModule.CalculatePositions(hand, positionConfiguration);
```

Comment:
This little trio of overwriting the same variable feels a little clunky (especially cos 2/3rds of the calls are in another object), could we pass a reference to the `ApplyAdditionalPositionModifiers` function into the positioning module call and have it do it all at once

Reply:
I could move that into the lower function, but there would be a similar setting of the variable in the positioningModule class. Ideally I think we would make the Positioning Module handle the PositionModifiers but then we would need a way to toggle the modifiers to use depending on the interaction which was a bit of additional complication that I think would be good to implement when we allow configuration of those items via settings